encrypt/encrypt.go

```diff
@@ -187,6 +193,27 @@ func checkFiles(files []helpers.EncryptionFileSet) error {
 		if helpers.FileExists(file.Encrypted) {
 			return fmt.Errorf("outfile %s already exists", file.Encrypted)
 		}
+
+		// Check if the input file is already encrypted
+		unEncryptedFile, err := os.Open(file.Unencrypted)
+		if err != nil {
+			return err
+		}
+		defer func() {
+			if err := unEncryptedFile.Close(); err != nil {
+				log.Errorf("Error closing file: %s\n", err)
+			}
+		}()
+
+		// Extracting the first 8 bytes of the header - crypt4gh
+		byteSlice := make([]byte, 8)
+		magicWord, err := unEncryptedFile.Read(byteSlice)
+		if err != nil {
+			return err
+		}
+		if string(byteSlice[0:magicWord]) == "crypt4gh" {
```

Comment:
no need to index `byteSlice`

```suggestion
		if string(byteSlice) == "crypt4gh" {
```

Reply:
Ok